accounts/migrations/0034_auto_20170908_1235.py

```diff
@@ -0,0 +1,213 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.2 on 2017-09-08 16:35
+
+from __future__ import unicode_literals
+import json
+from django.db import migrations
+
+children_created = 0
+demographic_data_created = 0
+
+def get_participant_json():
+    """
+    Loads JSON file with participants from Lookit v1
+    """
+    with open('participants.json') as data_file:
+        data = json.load(data_file)
+    return data
+
+def get_duplicate_emails(participants):
+    """
+    Some lookit v1 users have duplicate emails.  In new db, emails must be unique. Will need to manually migrate.
+    """
+    email_list = []
+    for participant in participants:
+        email_list.append(participant.get('attributes').get('email'))
+    return [email for email in email_list if email_list.count(email) > 1]
+
+def migrate_participants(apps, schema_editor):
+    """
+    Maps Lookit V1 participants to new db
+    """
+    participants = get_participant_json()
+    duplicates = get_duplicate_emails(participants)
+    print("Starting migration of old lookit participants...")
+    part_count = 0
+    copied = 0
+    for participant in participants:
+        part_count += 1
+        # Skip participants that have duplicate emails in db
+        if participant.get('attributes').get('email') not in set(duplicates):
+            print(f'Copying participant {part_count}/{len(participants)}')
+            create_participant(participant, apps)
+            copied += 1
+        else:
+            print(f'Skipping participant {part_count}/{len(participants)}')
+    print('========================================')
+    print(f'Total participants: {len(participants)}')
+    print(f'Skipped: {len(duplicates)} entries due to duplicate emails')
```

Comment:
It might be nice to maybe write a file with the records maybe `[{"email":"dawn@gmail.com", "guids": ["xyz12", "idk34"]}, ...]` or something like that.